Gems/DiffuseProbeGrid/Code/Source/Components/DiffuseProbeGridComponentController.cpp

```diff
@@ -352,6 +356,28 @@ namespace AZ
             m_featureProcessor->SetScrolling(m_handle, m_configuration.m_scrolling);
         }
 
+        void DiffuseProbeGridComponentController::SetFrameUpdateCount(uint32_t frameUpdateCount)
+        {
+            if (!m_featureProcessor)
```

Comment:
I wonder if it makes sense to apply these settings even if the FP isn't enabled so that after re-enabling it, the FP has the settings as last specified (as opposed to being a silent no-op as here)? If left as a no-op, maybe we should emit a warning at least to indicate a possible user error?

Reply:
This is the common pattern in all of the component controllers, mainly to support configurations that do not want a particular FP or Gem.  It's not necessarily an error since it could be perfectly viable to load a level containing the content and then ignore it when the FP/Gem is disabled.